release-matrix/release-matrix.json

```diff
@@ -4,6 +4,7 @@
     "k0rdentVersion": "v0.2.0-alpha4",
     "mkeOperatorImageVersion": "v1.3.2",
     "mkeOperatorChartVersion": "1.1.4",
+    "calicoOSSVersion": "v3.29.3",
```

Comment:
Just a thought: how about making calico nested under network.calico.version? This will allow us to add other CNIs later.
```
"network": {
      "calico": {
        "version": "v3.29.3"
      }
    },
```
what do you think?

Reply:
@sakshisharma84 that makes sense. I've updated it to a similar format like you suggested.